src/Extension/Resources/config/task/PbcCodesnifferFixerTask.yaml

```diff
@@ -1,25 +0,0 @@
----
```

Comment:
As far as I remember it was a bugfix. Are you sure we need to remove them?

Reply:
I fixed original command with path priority between project and sdk